uint8 botId
int32 ax
int32 ay
int32 az
int32 wx
int32 wy
int32 wz